hash/Cargo.toml

```diff
@@ -10,3 +10,5 @@ clap = "3.0.0-beta.2"
 
 hash-parser = {path = "../hash-parser" } # @@TODO: remove this when the typchecker is ready since it will be the one who uses this!
```

Comment:
Is this accurate?

Reply:
I thought this would be the case since we can call the hash-transform crate to get IR and then direct it to whatever backend

Reply:
Yeah but surely we have to provide `hash-transform` with the AST right? It won't call the parser by itself.

Reply:
Oh yes, it won't call the parser by itself, the frontend will call transform and then that calls parser, ast, type check, etc.